src/main/java/org/fundacionjala/at15/katas/pokerhands/adrian/Score.java

```diff
@@ -0,0 +1,145 @@
+package org.fundacionjala.at15.katas.pokerhands.adrian;
+
+public class Score {
+    private int score1 = 0;
+    private int score2 = 0;
+    private final int twoPairPoints = 3;
+    private final int threeKindPoints = 4;
+    private final int straightPoints = 5;
+    private final int flushPoints = 10;
+    private final int fullHousePoints = 20;
+    private final int fourOfAKindPoints = 30;
+    private final int straightFlushPoints = 100;
+    public Score(String hand1, String hand2) {
```

Comment:
The way you separated the logic for each hand is good, however, you should think a way to leverage your classes hierarchy to reduce the heavy constructor you have here